README.md

```diff
@@ -115,6 +119,50 @@ Commit and push these changes to your local clone on GitHub, and then [create a
 
 We will be notified automatically when you have created your pull request, and providing that everything adheres to nf-core guidelines we will endeavour to approve your pull request as soon as possible.
 
+## Adding a new pipeline specific config
+
+Follow the previous guidelines to create a nf-core/configs, update the pipeline specific documentation, and create the necessary files.
+Replace `<PIPELINE>` with the pipeline name for all the instructions in this guidelines.
+In a similar manner, replace `<PROFILE>` by the name of the profile.
+
+### Pipeline specific documentation
+
+Currently documentation is available for the following pipelines within the specific profile:
+
+* sarek
+  * [MUNIN](docs/pipeline/sarek/munin.md)
+
+### Enabling the specific configs within a specific pipeline
+
+> This has to be done on a fork of the `nf-core/<PIPELINE>` repository.
+
+If not already present, add to `nextflow.config`
```

Comment:
```suggestion
If not already present, add to the `nextflow.config` in the repository of the given pipeline itself.
```

Reply:
Isn't that comment clear enough?
 > This has to be done on a fork of the `nf-core/<PIPELINE>` repository.

Reply:
Maybe I should add a :warning: emoticon?

Reply:
Oops yes you're correct - the quote block is sufficient. 

I'm not used to warnings in quote blocks coming as the **first** bit of information of a section (I read quote blocks as 'after the fact' warnings, so I think I missed it as it was under the title).

Either removing the quote block (and replace in bold maybe) or add the emoticon, as you prefer.

Reply:
I'll do both